components/cdc/src/delegate.rs

```diff
@@ -590,7 +645,8 @@ impl Delegate {
             ..Default::default()
         };
         let send = move |downstream: &Downstream| {
-            if !downstream.state.load().ready_for_change_events() {
+            // No ready downstream or a dowsream that does not match the kv_api type, will be ignored.
+            if !downstream.state.load().ready_for_change_events() || downstream.kv_api != kv_api {
```

Comment:
Why raw kv events can be sent to downstreams even if the snapshot is not taken? Can you describe it somewhere?

Reply:
emm...For one delegate, it may contain rawkv downstreams and tidb downstreams. Here I want to filter tidb downstreams when send rawkv events.